acceptance/bundle/telemetry/deploy-compute-type/output.txt

```diff
@@ -0,0 +1,214 @@
+
+>>> [CLI] bundle deploy -t one
+Uploading bundle files to /Workspace/Users/[USERNAME]/.bundle/deploy-compute-type/one/files...
+Deploying resources...
+Updating deployment state...
+Deployment complete!
+
+>>> [CLI] bundle deploy -t two
+Uploading bundle files to /Workspace/Users/[USERNAME]/.bundle/deploy-compute-type/two/files...
+Deploying resources...
+Updating deployment state...
+Deployment complete!
+
+>>> [CLI] bundle deploy -t three
+Uploading bundle files to /Workspace/Users/[USERNAME]/.bundle/deploy-compute-type/three/files...
+Deploying resources...
+Updating deployment state...
+Deployment complete!
+
+>>> [CLI] bundle deploy -t four
+Uploading bundle files to /Workspace/Users/[USERNAME]/.bundle/deploy-compute-type/four/files...
+Deploying resources...
+Updating deployment state...
+Deployment complete!
+
+>>> cat out.requests.txt
+{
+  "bundle_uuid": "[UUID]",
+  "deployment_id": "[UUID]",
+  "resource_count": 1,
+  "resource_job_count": 1,
+  "resource_pipeline_count": 0,
+  "resource_model_count": 0,
+  "resource_experiment_count": 0,
+  "resource_model_serving_endpoint_count": 0,
+  "resource_registered_model_count": 0,
+  "resource_quality_monitor_count": 0,
+  "resource_schema_count": 0,
+  "resource_volume_count": 0,
+  "resource_cluster_count": 0,
+  "resource_dashboard_count": 0,
+  "resource_app_count": 0,
+  "resource_job_ids": [
+    "1"
+  ],
+  "experimental": {
+    "configuration_file_count": 1,
+    "variable_count": 0,
+    "complex_variable_count": 0,
+    "lookup_variable_count": 0,
+    "target_count": 4,
+    "bool_values": [
```

Comment:
should we filter out bool_values only? I assume other tests cover the rest of this struct. (easier to review and less noise on change to common fields).